packages/fast-components-react-msft/package.json

```diff
@@ -90,6 +90,7 @@
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
     "eyes.images": "3.6.2",
+    "exenv-es6": "^1.0.0",
```

Comment:
You've added this as a devDependency but you are using it as an export, you should add it to the peerDependencies list as well. I believe we have other packages that use it as a peerDependency but we shouldn't rely on that.